documentation/book/con-custom-resources-status.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// assembly-custom-resources.adoc
+
+[id='con-custom-resources-status-{context}']
+
+= {ProductName} custom resource status
+
+The `status` property of a {ProductName}-specific custom resource publishes the _current_ state of the resource to users and tools that need the information.
+
+Status information is useful for tracking progress related to a resource achieving its _desired_ state, as defined by the `spec` property. The `status` provides the time and reason the state of the resource changed and details of events preventing or delaying the Operator from realizing the desired state.
+
+{ProductName} creates and maintains the status of custom resources, periodically evaluating the current state of the custom resource and updating its status accordingly.
```

Comment:
Should we mention here that:
* The status is not editable when changing the CR it self (e.g. using `kubectl edit`)
* That even changing the status would not affect / reconfigure the Kafka cluster?

Reply:
When performing an update on a custom resource using `{kubectl} edit`, for example, its `status` is not editable. Moreover, changing the `status` would not affect the configuration of the Kafka cluster.